R/scale-.R

```diff
@@ -717,6 +723,12 @@ ScaleContinuous <- ggproto("ScaleContinuous", Scale,
       return(numeric())
     }
     transformation <- self$get_transformation()
+    breaks <- self$breaks %|W|% transformation$breaks
```

Comment:
I like to think of this line as the 'spirit of this PR'. We no longer need to treat function-breaks and default breaks any differently.